modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -73,6 +73,7 @@ export interface NotificationTemplate {
 	recipientType: RecipientType;
```

Comment:
Change [this commit](https://github.com/seltong/liferay-portal/pull/323/commits/05ef8d8cc91b950052dcafdc65a275338b866ec3) message to `Add system parameter in Notification Template`